web/src/platform/context.ts

```diff
@@ -80,7 +80,9 @@ export function createPlatformContext(): PlatformContext {
 function toPrettyWebBlobURL(
     ctx: RepoFile & Partial<UIPositionSpec> & Partial<ViewStateSpec> & Partial<UIRangeSpec> & Partial<RenderModeSpec>
 ): string {
-    return toPrettyBlobURL(ctx, { isWebURL: true })
+    const urlSearchParams = new URLSearchParams(toPrettyBlobURL(ctx))
+    urlSearchParams.set('subtree', 'true')
+    return `${decodeURIComponent(urlSearchParams.toString())}`
```

Comment:
This is passing a relative URL including the path to `URLSearchParams`, which only takes a search part of the URL. The decoding also looks wrong to me. This should work I think (untested):
```suggestion
    const url = new URL(toPrettyBlobURL(ctx), location.href)
    url.searchParams.set('subtree', 'true')
    return url.pathname + url.search + url.hash
```